cmd/neofs-node/morph.go

```diff
@@ -37,6 +36,10 @@ func initMorphComponents(c *cfg) {
 	var err error
 
 	fn := func(addresses []string, dialTimeout time.Duration, handler func(*client.Client)) {
```

Comment:
isnt the `fn` var redundant now?

Reply:
A little bit, but I left the option to easily revert connection to mainchain. For the same reason, the values in the config have not yet been cut.